support/ebpf/native_stack_trace.ebpf.c

```diff
@@ -107,60 +110,82 @@ bool bsearch_step(void* inner_map, u32* lo, u32* hi, u16 page_offset) {
 }
 
 // Get the outer map based on the number of stack delta entries.
-static inline __attribute__((__always_inline__))
-void *get_stack_delta_map(int mapID) {
+static inline __attribute__((__always_inline__)) void *get_stack_delta_map(int mapID)
+{
   switch (mapID) {
-  case  8: return &exe_id_to_8_stack_deltas;
-  case  9: return &exe_id_to_9_stack_deltas;
-  case 10: return &exe_id_to_10_stack_deltas;
-  case 11: return &exe_id_to_11_stack_deltas;
-  case 12: return &exe_id_to_12_stack_deltas;
-  case 13: return &exe_id_to_13_stack_deltas;
-  case 14: return &exe_id_to_14_stack_deltas;
-  case 15: return &exe_id_to_15_stack_deltas;
-  case 16: return &exe_id_to_16_stack_deltas;
-  case 17: return &exe_id_to_17_stack_deltas;
-  case 18: return &exe_id_to_18_stack_deltas;
-  case 19: return &exe_id_to_19_stack_deltas;
-  case 20: return &exe_id_to_20_stack_deltas;
-  case 21: return &exe_id_to_21_stack_deltas;
-  case 22: return &exe_id_to_22_stack_deltas;
-  case 23: return &exe_id_to_23_stack_deltas;
-  default: return NULL;
+  case 8:
+    return &exe_id_to_8_stack_deltas;
+  case 9:
+    return &exe_id_to_9_stack_deltas;
+  case 10:
+    return &exe_id_to_10_stack_deltas;
+  case 11:
+    return &exe_id_to_11_stack_deltas;
+  case 12:
+    return &exe_id_to_12_stack_deltas;
+  case 13:
+    return &exe_id_to_13_stack_deltas;
+  case 14:
+    return &exe_id_to_14_stack_deltas;
+  case 15:
+    return &exe_id_to_15_stack_deltas;
+  case 16:
+    return &exe_id_to_16_stack_deltas;
+  case 17:
+    return &exe_id_to_17_stack_deltas;
+  case 18:
+    return &exe_id_to_18_stack_deltas;
+  case 19:
+    return &exe_id_to_19_stack_deltas;
+  case 20:
+    return &exe_id_to_20_stack_deltas;
+  case 21:
+    return &exe_id_to_21_stack_deltas;
+  case 22:
+    return &exe_id_to_22_stack_deltas;
+  case 23:
+    return &exe_id_to_23_stack_deltas;
+  default:
+    return NULL;
```

Comment:
Can we somehow keep the prior formatting?

Reply:
Added and applied AllowShortCaseLabelsOnASingleLine with https://github.com/open-telemetry/opentelemetry-ebpf-profiler/pull/310/commits/5cc31f5d0f3ea94e053484513b87282a2cb91a51